background/style-manager.js

```diff
@@ -378,12 +378,18 @@ const styleMan = (() => {
             for (const {style: someStyle} of dataMap.values()) {
               if (someStyle._id === style._id) {
                 someStyle._isUswLinked = true;
-                someStyle.sourceCode = style.sourceCode;
-                const {metadata} = await API.worker.parseUsercssMeta(style.sourceCode);
+                someStyle.originalValue = style.sourceCode;
+                let metadata = {};
+                try {
+                  const {metadata: tmpMetadata} = await API.worker.parseUsercssMeta(style.sourceCode);
+                  metadata = tmpMetadata;
+                } catch (err) {
+                  console.log(err);
+                }
                 someStyle.metadata = metadata;
               } else {
                 delete someStyle._isUswLinked;
-                delete someStyle.sourceCode;
+                delete someStyle.originalValue;
```

Comment:
`originalValue` is overly nonspecific. It should reflect its purpose and relation to the value it contains. Something like `tmpSourceCode`. Actually it might be better not to add any temporary properties at all but rather use an object declared at start of style-manager.js keyed by style id.